projects/cds/src/cms-components/merchandising/carousel/merchandising-carousel/merchandising-carousel.component.html

```diff
@@ -0,0 +1,20 @@
+<cx-carousel
+  [items]="items$ | async"
+  [title]="title$ | async"
+  [template]="merchandisingCarouselItem"
+  itemWidth="285px"
+>
+</cx-carousel>
+
+<ng-template #merchandisingCarouselItem let-item="item">
+  <a tabindex="0" [routerLink]="{ cxRoute: 'product', params: item } | cxUrl">
```

Comment:
where's the metadata for the product level?

Reply:
This will be done in ticket https://jira.hybris.com/browse/COL-164, which will be in the next sprint starting next week